packages/stencil/src/components/datacom-radio/datacom-radio.css

```diff
@@ -49,124 +47,142 @@
 }
 
 /* Label with standard radio */
-.dc-radio-size-standard + label {
+.dc-radio-size-standard .dc-radio-label {
   font-size: 16px;
 }
 
 /* Label with small radio */
-.dc-radio-size-small + label {
+.dc-radio-size-small .dc-radio-label {
   font-size: 14px;
 }
 
-.dc-radio-radios:hover {
+.dc-radio-radios .dc-radio-input:hover {
   border: 1px solid var(--dc-primary-colour);
   border-radius: 12px;
 }
 
-.dc-radio-radios:focus {
+.dc-radio-radios .dc-radio-input:focus {
   outline: 3px solid var(--dc-primary-colour);
   border-radius: 12px;
   border: 1px solid var(--dc-greyscale-05);
   outline-offset: 1px;
 }
 
-.dc-radio-radios:focus:not(:focus-visible) {
+.dc-radio-radios .dc-radio-label:focus-visible {
   outline: none;
 }
 
-.dc-radio-radios:checked {
+.dc-radio-buttons .dc-radio-label:focus-visible {
+  outline: 3px solid var(--dc-primary-colour);
+  border: 1px solid var(--dc-greyscale-05);
+  outline-offset: 1px;
+}
+.dc-radio-grouped .dc-radio-label:focus-visible {
+  outline: 3px solid var(--dc-primary-colour);
+  border: 1px solid var(--dc-greyscale-05);
+  outline-offset: -2px;
+  gap: 4px;
+}
+.dc-radio-radios .dc-radio-input:checked {
   border: 3px solid var(--dc-background-colour);
   background: var(--dc-primary-colour);
   box-shadow: 0 0 0 1px var(--dc-primary-colour);
   outline: none;
   transition: opacity 0.0005s ease-in-out;
 }
-
-.dc-radio-radios:disabled {
+.dc-radio-radios .dc-radio-input:disabled {
   background-color: var(--dc-background-accent-color);
   border: 1px solid var(--dc-greyscale-05);
 }
 
-.dc-radio-disabled + label {
+.dc-radio-disabled .dc-radio-label {
   color: var(--dc-greyscale-03);
 }
 
-.dc-radio-disabled.dc-radio-radios:checked {
+.dc-radio-disabled.dc-radio-radios .dc-radio-input:checked {
   border: 3px solid var(--dc-background-accent-color);
   background: var(--dc-greyscale-04);
   box-shadow: 0 0 0 1px var(--dc-greyscale-05);
 }
 
-/* main containter  */
-
-.dc-radio-buttons {
+/* defalut radio button  */
+/* For grouped */
+.dc-radio-grouped .dc-radio-input,
+.dc-radio-buttons .dc-radio-input {
   position: absolute;
   cursor: pointer;
-  flex-direction: row;
-  opacity: 0;
-  width: 100px;
-  height: 60px;
+  /* flex-direction: row; */
+  display: none;
   overflow: hidden;
 }
+/*  */
 
-.dc-radio-buttons + label {
+.dc-radio-buttons .dc-radio-input + label {
   border: 2px solid var(--dc-greyscale-05);
   display: flex;
   background-color: var(--dc-background-colour);
   font-family: var(--dc-font-family);
   font-weight: 500;
   align-items: center;
-  margin-left: 12px;
   border-radius: 8px;
+  margin-left: 12px;
+}
+.dc-radio-grouped .dc-radio-input + label {
+  border: 1px solid var(--dc-greyscale-05);
+  display: flex;
+  background-color: var(--dc-background-colour);
+  font-family: var(--dc-font-family);
+  font-weight: 500;
+  align-items: center;
 }
 
-.dc-radio-buttons.dc-radio-size-standard + label {
+/* for standar size button and */
```

Comment:
Misspelt the word "standard". Also you ended the sentence with an 'and'? Is this comment incomplete?

Reply:
Fixed it.